packages/pluginsdk-peerdeps/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@spinnaker/pluginsdk-peerdeps",
+  "description": "Provides package dependencies to plugin developers",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "scripts": {
+    "usage": "cat USAGE.txt",
+    "stage": "npm run clean && ./sync-stage.sh",
+    "sync": "./sync-save.js",
+    "clean": "./clean.sh"
+  },
+  "files": [],
+  "peerDependencies": {
+    "@spinnaker/core": "0.0.507",
+    "@spinnaker/pluginsdk": "0.0.21",
+    "@uirouter/core": "6.0.4",
+    "@uirouter/react": "1.0.2",
+    "lodash-es": "4.17.15",
+    "prop-types": "15.6.1",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "rxjs": "5.4.2",
+    "@rollup/plugin-commonjs": "15.1.0",
+    "@rollup/plugin-node-resolve": "9.0.0",
+    "@rollup/plugin-typescript": "6.0.0",
+    "@spinnaker/eslint-plugin": "1.0.8",
+    "@types/react": "16.8.25",
+    "@typescript-eslint/eslint-plugin": "4.4.0",
+    "@typescript-eslint/parser": "4.4.0",
+    "bufferutil": "4.0.1",
+    "eslint": "7.10.0",
+    "eslint-config-prettier": "6.12.0",
+    "eslint-plugin-react-hooks": "4.1.2",
+    "husky": "4.3.0",
+    "npm-run-all": "4.1.5",
+    "prettier": "2.1.2",
+    "pretty-quick": "3.0.2",
+    "rollup": "2.28.2",
+    "rollup-plugin-external-globals": "0.6.0",
+    "rollup-plugin-less": "1.1.2",
+    "rollup-plugin-postcss": "3.1.8",
+    "shx": "0.3.2",
+    "typescript": "4.0.2",
+    "utf-8-validate": "5.0.2"
+  },
+  "peerDevDependencies.doc": "These will be installed as devDependencies by check-peer-dependencies --install",
+  "peerDevDependencies": [
+    "@rollup/plugin-commonjs",
+    "@rollup/plugin-node-resolve",
+    "@rollup/plugin-typescript",
+    "@spinnaker/eslint-plugin",
+    "@types/react",
+    "@typescript-eslint/eslint-plugin",
+    "@typescript-eslint/parser",
+    "bufferutil",
```

Comment:
This list disambiguates the regular vs devDependencies in the list of peer dependencies above.